### ISSUE TITLE
Fix manifest URL rewriting

### ISSUE THREAD

**Issue by jon-betts** [hypothesis/via#288]
We don't rewrite URLs in `manifest.json` files: https://twitter.com/bascule/status/1307440596668182528?ref_src=twsrc%5Etfw%7Ctwcamp%5Etweetembed%7Ctwterm%5E1307440596668182528%7Ctwgr%5Eshare_3&ref_url=https%3A%2F%2Fwww.theguardian.com%2Ftechnology%2F2020%2Fsep%2F21%2Ftwitter-apologises-for-racist-image-cropping-algorithm

This causes problems for certain websites. We have actually taken steps to prevent `pywb` from rewriting manifest files at all, as it was adding all the javascript stuff to some of them.

Long story short, this might not be possible, but could enable some big sites like twitter.
